uportal-war/src/main/java/org/jasig/portal/groups/pags/testers/BaseAttributeTester.java

```diff
@@ -26,48 +28,60 @@
  * @author Dan Ellentuck
  * @version $Revision$
  */
-
 public abstract class BaseAttributeTester implements IPersonTester {
-    protected String attributeName;
-    protected String testValue;
+    protected final String attributeName;
+    protected final String testValue;
```

Comment:
Stops being immutable or mutable?

I'm not sure subclasses should or will need to change the values of those member variables to do their work.  I tend to put `final` where I can.  But this part of the change is just something I threw in because I noticed they were never being re-set by subclasses... it's not a critical part of it.